app/Enums/Weekday.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Enums;
+
+enum Weekday: string
+{
+    case Monday = 'monday';
+    case Tuesday = 'tuesday';
+    case Wednesday = 'wednesday';
+    case Thursday = 'thursday';
+    case Friday = 'friday';
+    case Saturday = 'saturday';
+    case Sunday = 'sunday';
+
+    public static function labels(): array
+    {
+        return [
+            'monday' => __('Monday'),
+            'tuesday' => __('Tuesday'),
+            'wednesday' => __('Wednesday'),
+            'thursday' => __('Thursday'),
+            'friday' => __('Friday'),
+            'saturday' => __('Saturday'),
+            'sunday' => __('Sunday'),
```

Comment:
I think these may have been included in the strings placeholder file. If so, they could be removed from there now.